analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/PermissiveCors.Net.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.AspNetCore.Cors.Infrastructure;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Net.Http.Headers;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Primitives;
+
+namespace Tests.Diagnostics
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class CorsEnabledManualAddedHeadersController : Controller
+    {
+        private const string Star = "*";
+
+        [HttpGet]
+        public void Index(string header, string headerValue)
+        {
+            // The Access-Control-Allow-Origin response header indicates whether the response can be shared with requesting code from the given origin.
+            Response.Headers.Add("Access-Control-Allow-Origin", "*"); // Noncompliant
+            Response.Headers.Add("Access-Control-Allow-Origin", Star); // Noncompliant
+            Response.Headers.Add("Access-Control-Allow-Origin", "https://trustedwebsite.com");
```

Comment:
could you also add a case `Response.Headers.Add("Access-Control-Allow-Origin", "*****");` ?